client/components/profile/profile-category.component.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const PROFILE_CATEGORY_COMPONENT_NAME = 'profileCategory';
+
+const profileCategoryComponent = {
+  bindings: {
+    options: '<',
+    profile: '<'
+  },
+  templateUrl: 'components/profile/profile-category.html',
+  controllerAs: 'profileCategoryCtrl',
+  controller: class demoController {
```

Comment:
Ah exact c'est copié collé de ici https://medium.com/@erwan.datin/angular-1-5-component-in-es6-my-way-to-write-it-as-near-as-possible-as-an-angular2-component-b0bbd9f6ffaa